[17.0][ADD] Add pos_partner_birthdate_warning module

parvezqureshi opened the pull request.

Commit:
[ADD] Add pos_cash_move_reason module

legalsylvain committed:
[FIX] make description working in odoo / appstore

legalsylvain committed:
new module pos_margin

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-transbot committed:
OCA Transbot updated translations from Transifex

legalsylvain committed:
[ADD] demo data
[REF] factor code
[UPD] fr translation and pot file

oca-transbot committed:
OCA Transbot updated translations from Transifex

legalsylvain committed:
[8.0] [ADD] pos order to sale order (w/o picking) (#188)

* [ADD] new module pos_order_to_sale_order

legalsylvain committed:
[FIX] taxes was dropped in the sale order lines. [IMP] Remove limitation of this module (price unit loss, and pricelist) if pos_pricelist is installed

legalsylvain committed:
[BUMP] version

Commit:
Coquille + ajout (BL) pour boutons "vente livrée"

pedrobaeza committed:
[FIX] por_order_to_sale_order: PEP8

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

Commit:
[MIG] pos_margin:Migration to 11.0

Commit:
[IMP] Add pos margin in pos order report

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

legalsylvain committed:
[FIX] do not allow to create sale order several times;
[FIX] Remove console.log ;
[IMP] Do not import js files in backend
[IMP] apply new readme structure
[FIX] remove useless demo file
[FIX] some pylint issues
[REF] add maintainers and developpment status

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-8.0/pos-8.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-8-0/pos-8-0-pos_order_to_sale_order/

legalsylvain committed:
[MIG][12.0] port pos_cash_move_reason
[REF] Don't use ``product.product`` model for Reasons, because Odoo remove
the fields ``expense_pdt`` ``and income_pdt`` from the model.
Use instead a new model ``pos.move.reason`` for this purpose.
[REF] Doesn't inherit from ``cash.box.in`` and ``cash.box.out`` model,
as there are bad designed and doesn't allow clean inheritance.
Instead, use new transient model ``wizard.pos.move.reason``.
[REF] Oca convention. add readme folder.

legalsylvain committed:
[MIG] pos_margin: Migration to 12.0

meodoo committed:
[MIG] pos_margin: Migration to 13.0

legalsylvain committed:
[REF] pos_order_to_sale_order: Black python code

legalsylvain committed:
[MIG] pos_order_to_sale_order: Migration to 12.0

OCA-git-bot committed:
pos_cash_move_reason 12.0.3.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_cash_move_reason/

eLBati committed:
ADD pos_user_restriction
Restrict some users to only access their assigned points of sale

legalsylvain committed:
[FIX] remove commented code

Co-authored-by: Iván Todorovich <ivan.todorovich@druidoo.io>

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
pos_order_to_sale_order 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_order_to_sale_order/

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-12.0/pos-12.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_cash_move_reason/es/

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: pos-12.0/pos-12.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_order_to_sale_order/es/

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: pos-12.0/pos-12.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_user_restriction/es/

eLBati committed:
FIX pos_user_restriction allowing "User: Assigned POS Only" to see point of sale menu

OCA-git-bot committed:
pos_user_restriction 12.0.1.0.1

augustodinizl committed:
Added translation using Weblate (Portuguese (Brazil))

augustodinizl committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (23 of 23 strings)

Translation: pos-12.0/pos-12.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_order_to_sale_order/pt_BR/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_cash_move_reason/

legalsylvain committed:
[FIX] pos_cash_move_reason : date of statement line depends now on the context timezone

OCA-git-bot committed:
pos_cash_move_reason 12.0.3.0.2

gaikaz committed:
pos_order_to_sale_order: Make JS extendable

Return module's javascript widgets to make them extendable for other modules.

Same idea as #535

OCA-git-bot committed:
pos_order_to_sale_order 12.0.1.0.2

legalsylvain committed:
[FIX] pos_order_to_sale_order : dependencies to sale module is now required in recent odoo version

legalsylvain committed:
[FIX] pos_order_to_sale_order : make the module working if get_order() is null, that occures if pos_restaurant is installed

jguenat committed:
Added translation using Weblate (French (Switzerland))

jguenat committed:
Translated using Weblate (French (Switzerland))

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-12.0/pos-12.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_cash_move_reason/fr_CH/

marylla committed:
Translated using Weblate (German)

Currently translated at 7.9% (3 of 38 strings)

Translation: pos-12.0/pos-12.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_cash_move_reason/de/

legalsylvain committed:
[FIX] pos_user_restriction: disable test that are failing if pos_restaurant module is installed

OCA-git-bot committed:
pos_user_restriction 12.0.1.0.2

acsonefho committed:
[IMP] pos_cash_move_reason: black, isort, prettier

acsonefho committed:
[13.0][MIG] pos_cash_move_reason

dsolanki-initos committed:
[MIG] Migrate module pos_cash_move_reason to v14.

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

OCA-git-bot committed:
pos_order_to_sale_order 12.0.1.0.3

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_order_to_sale_order/

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_order_to_sale_order/

hkapatel-initos committed:
[IMP]pos_user_restriction: black, isort, prettier

hkapatel-initos committed:
[MIG] pos_user_restriction: Migration to 14.0

dsolanki-initos committed:
[IMP] pos_margin: isort, black, prettier.

dsolanki-initos committed:
[MIG] Migrate module pos_margin to v14.

legalsylvain committed:
[IMP] pos_margin : display margin and margin rate in front-end UI

Co-authored-by: Iván Todorovich <ivan.todorovich@gmail.com>

manuelregidor committed:
[IMP] pos_order_to_sale_order: black, isort, prettier

manuelregidor committed:
[MIG]pos_order_to_sale_order: Migration to 13.0

oca-travis committed:
[UPD] Update pos_order_to_sale_order.pot

OCA-git-bot committed:
[UPD] README.rst

sbidoul committed:
pos_margin: bump version to workround PyPI glitch

acsonefho committed:
[14.0][FIX] pos_cash_move_reason: update/fix ir.rule multi-company

OCA-git-bot committed:
pos_cash_move_reason 14.0.1.0.1

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

Commit:
[FIX] pos_cash_move_reason:
- Changed the field where the reason for the withdrawal or receipt of money is indicated, if left in the field name, it changes the name of the accounting entry and gives an error.

hailangvn committed:
[ADD] pos_hide_cost_price_and_margin: Add to 15.0

This module is to hide cost, margin, and replenishment info from product
detail in pos. It utilize a setting from pos_margin module but doesn't
need to depend on the later module thanks to the mechanism that if there
is no field iface_display_margin, the value of t-if is False.

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-14.0/pos-14.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_cash_move_reason/ca/

Commit:
[UPD] Update pos_hide_cost_price_and_margin.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
pos_hide_cost_price_and_margin 15.0.1.0.1

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 37.5% (3 of 8 strings)

Translation: pos-14.0/pos-14.0-pos_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_margin/it/

OCA-git-bot committed:
pos_cash_move_reason 14.0.1.0.2

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-14.0/pos-14.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_cash_move_reason/it/

Commit:
[IMP] allow to hide the variants in a product template selection

This can help perfs when there are templates with a lot of variants.

Also this adds a "confirm" button that can be a practical alternative to
the variant selection.

AshishHirapara committed:
[14.0][FIX] pos_cash_move_reason: Admin access issue for pos actions

OCA-git-bot committed:
pos_cash_move_reason 14.0.1.0.3

cvinh committed:
[FIX] Hide margin in Front End if config is false

OCA-git-bot committed:
pos_margin 14.0.1.0.2

legalsylvain committed:
[PORT] pos_order_to_sale_order from 13.0 to 16.0:
- REF : Move pos.config configuration into res.config.settings
- REF : adapt JS and XML code to new OWL framework
- REF : Adapt python code to removal of onchange (simplification)
- IMP : Add action icons
- ADD : Add 'invoiced 'action
- IMP : Append PoS customer note in the sale order line name field
- IMP : Add tests

AshishHirapara committed:
[IMP][14.0] pos_user_restriction

Updated roadmap description

Commit:
[UPD] Update pos_order_to_sale_order.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/

Commit:
[MIG] pos_cash_move_reason: Migration to 15.0

ygcarvalh committed:
[IMP] pos_product_template: Sort product attribute by sequence

luismalta committed:
[IMP] Allows SelectVariantPopup to be updated when chained to itself

Signed-off-by: Luis Malta <luis.malta@kmee.com.br>

Commit:
[UPD] Update pos_order_to_sale_order.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/

Commit:
[UPD] Update pos_product_template.pot

OCA-git-bot committed:
pos_product_template 14.0.1.0.2

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-14.0/pos-14.0-pos_product_template
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_product_template/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/it/

legalsylvain committed:
[FIX] pos_order_to_sale_order

set correct image path in the description of the module.

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
pos_order_to_sale_order 16.0.1.0.2

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 37.5% (3 of 8 strings)

Translation: pos-14.0/pos-14.0-pos_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_margin/it/

suker committed:
[IMP] : pre-commit stuff

suker committed:
[MIG] pos_margin: Migration to 16.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: pos-15.0/pos-15.0-pos_hide_cost_price_and_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_hide_cost_price_and_margin/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (9 of 9 strings)

Translation: pos-14.0/pos-14.0-pos_product_template
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_product_template/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/it/

FLNerea committed:
Translated using Weblate (Spanish)

Currently translated at 66.6% (14 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/it/

Commit:
[MIG] pos_user_restriction: Migration to 15.0

Commit:
Translated using Weblate (Catalan)

Currently translated at 97.3% (37 of 38 strings)

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/ca/

Commit:
[UPD] Update pos_user_restriction.pot

OCA-git-bot committed:
[UPD] README.rst

geomer198 committed:
[FIX] pos_order_to_sale_order: Added return for CreateOrderButton component.

OCA-git-bot committed:
pos_order_to_sale_order 16.0.1.0.3

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (38 of 38 strings)

Translation: pos-15.0/pos-15.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_cash_move_reason/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Translation: pos-15.0/pos-15.0-pos_hide_cost_price_and_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_hide_cost_price_and_margin/es/

Commit:
[UPD] Update pos_margin.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
pos_margin 16.0.1.0.1

OCA-git-bot committed:
pos_product_template 14.0.1.0.3

anmarmo1 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: pos-14.0/pos-14.0-pos_product_template
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_product_template/es/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: pos-16.0/pos-16.0-pos_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_margin/it/

GabbasovDinar committed:
[IMP] pos_order_to_sale_order: Improvement for the ability to use the result of sales order creation after inheritance

OCA-git-bot committed:
pos_order_to_sale_order 16.0.1.0.4

GabbasovDinar committed:
[16.0][REF] pos_order_to_sale_order: create method optimization

OCA-git-bot committed:
pos_order_to_sale_order 16.0.1.0.5

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/es/

lbpatri committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/es/

lbpatri committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/es/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

legalsylvain committed:
[FIX] pos_order_to_sale_order : make test compatible with pos_minimize_menu

Commit:
[IMP] pos_product_template: black, isort, prettier

Commit:
[MIG] pos_product_template: Migration to 15.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-15.0/pos-15.0-pos_hide_cost_price_and_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_hide_cost_price_and_margin/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-15.0/pos-15.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_user_restriction/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: pos-15.0/pos-15.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_user_restriction/it/

AntoniRomera committed:
[MIG] pos_cash_move_reason: Migration to 16.0

chienandalu committed:
[FIX] *: Linter fixes

legalsylvain committed:
[REF] pos_order_to_sale_order : add sale order line sequence in the prepare function

legalsylvain committed:
[IMP] pos_order_to_sale_order : use customer_note to prepare sale order line name fields

legalsylvain committed:
[IMP] pos_order_to_sale_order : performance : directly overload _get_sale_order_line_multiline_description_sale to avoid to write on sale.order.line after having creating the sale.order.line

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: pos-15.0/pos-15.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-15-0/pos-15-0-pos_user_restriction/es/

OCA-git-bot committed:
[BOT] post-merge updates

geomer198 committed:
[IMP] pos_order_to_sale_order: Code is refactored and tests are updated.

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update pos_product_template.pot

etobella committed:
[ADD] pos_order_copy

legalsylvain committed:
[FIX] eslint error 'no-shadow' in all modules

OCA-git-bot committed:
[BOT] post-merge updates

luisDIXMIT committed:
[IMP] pos_product_template: pre-commit auto fixes

luisDIXMIT committed:
[MIG] pos_product_template: Migration to 17.0

etobella committed:
[MIG] pos_product_template: Finish migration

trisdoan committed:
[IMP] pos_hide_cost_price_and_margin: pre-commit auto fixes

trisdoan committed:
[MIG] pos_hide_cost_price_and_margin: Migration to 17.0

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (21 of 21 strings)

Translation: pos-16.0/pos-16.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_order_to_sale_order/pt_BR/

ivantodorovich committed:
[IMP] pos_order_to_sale_order: use test helpers

OCA-git-bot committed:
[BOT] post-merge updates

traitrantrobz committed:
[IMP] pos_cash_move_reason: pre-commit auto fixes

chaule97 committed:
[IMP] pos_order_to_sale_order: pre-commit auto fixes

chaule97 committed:
[MIG] pos_order_to_sale_order: Migration to 17.0

traitrantrobz committed:
[MIG] pos_cash_move_reason: Migration to 17.0

etobella committed:
Update copier: Add rebel module for pos_lot_barcode

Commit:
[UPD] Update pos_product_template.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-17.0/pos-17.0-pos_product_template
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_product_template/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: pos-17.0/pos-17.0-pos_product_template
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_product_template/it/

Commit:
[UPD] Update pos_order_to_sale_order.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[17.0][ADD] pos_partner_vat_required: New module pos_partner_vat_required

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-17.0/pos-17.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_order_to_sale_order/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (22 of 22 strings)

Translation: pos-17.0/pos-17.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_order_to_sale_order/it/

Commit:
[17.0][ADD] pos_partner_address_required: New module pos_partner_address_required

Commit:
[17.0][ADD] pos_partner_firstname_required: New module pos_partner_firstname_required

Commit:
[UPD] Update pos_hide_cost_price_and_margin.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: pos-17.0/pos-17.0-pos_hide_cost_price_and_margin
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_hide_cost_price_and_margin/it/

janbec committed:
Added translation using Weblate (German)

janbec committed:
Translated using Weblate (German)

Currently translated at 100.0% (7 of 7 strings)

Translation: pos-17.0/pos-17.0-pos_partner_firstname
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_partner_firstname/de/

janbec committed:
Added translation using Weblate (German)

janbec committed:
Translated using Weblate (German)

Currently translated at 100.0% (5 of 5 strings)

Translation: pos-17.0/pos-17.0-pos_lot_selection
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_lot_selection/de/

em230418 committed:
[IMP] pos_user_restriction: pre-commit auto fixes

em230418 committed:
[MIG] pos_user_restriction: migrate to 17.0

Migration notes:

1. Removed bypassing group check, since set_cashbox_pos in 16.0+
does not call has_groups method [1]

2. Removed hack in res_users.py, which should not be the case,
since in res.users' write method has_group is replaced with
_is_internal [2]

3. Removed several ir.model.access records for removed models:

- account.bank.statement.cashbox [3]
- account.cashbox.line [3]
- cash.box.out [4]

[1] https://github.com/odoo/odoo/pull/113151
[2] https://github.com/odoo/odoo/commit/dae28c4b463dde177b7ac807382bed0f4e0678c1#diff-16e37db365c1ea1f2e6c79aeb60c80f68f1a5c75970e8e3f198eb18af56278dd
[3] https://github.com/odoo/odoo/commit/4fb0933986534e31337aa1dce3478a67deea72de
[4] https://github.com/odoo/odoo/commit/7e23eda02fc00b4e4a85e3a71b14fb1081441a04

In this commit we remove bypassing group check

em230418 committed:
[IMP] pos_user_restriction: add tests for get_closing_control_data

Commit:
[UPD] Update pos_order_copy.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update pos_partner_vat_required.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: pos-17.0/pos-17.0-pos_partner_vat_required
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_partner_vat_required/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: pos-17.0/pos-17.0-pos_order_copy
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_order_copy/it/

Commit:
[UPD] Update pos_partner_firstname_required.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update pos_partner_address_required.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: pos-17.0/pos-17.0-pos_partner_firstname_required
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_partner_firstname_required/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: pos-17.0/pos-17.0-pos_partner_address_required
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_partner_address_required/it/

mtbochaca committed:
Added translation using Weblate (Catalan)

mtbochaca committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (22 of 22 strings)

Translation: pos-17.0/pos-17.0-pos_order_to_sale_order
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_order_to_sale_order/ca/

Commit:
[17.0][ADD] pos_cash_control_extension: New module pos_cash_control_extension

Commit:
[UPD] Update pos_cash_control_extension.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update pos_user_restriction.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-17.0/pos-17.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_user_restriction/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: pos-17.0/pos-17.0-pos_cash_control_extension
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_cash_control_extension/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: pos-17.0/pos-17.0-pos_user_restriction
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_user_restriction/it/

Commit:
[UPD] Update pos_cash_move_reason.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-17.0/pos-17.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_cash_move_reason/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (37 of 37 strings)

Translation: pos-17.0/pos-17.0-pos_cash_move_reason
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_cash_move_reason/it/

OCA-git-bot committed:
[IMP] update dotfiles

danielduqma committed:
[FIX] pos_order_to_sale_order: recompute taxes

Recompute `sale.order` taxes, as received ones are the ones from
product, without fiscal position mapping applied.

OCA-git-bot committed:
[BOT] post-merge updates

vehi-invitu committed:
[17.0][ADD] Add pos_partner_birthdate_warning module

cvinh reviewed (commented):
Why don't you simply load res_partner.age field that is already computed in partner_contact_birthdate instead of recompute it ? Is it because it's not stored ?

parvezqureshi commented:
> Why don't you simply load res_partner.age field that is already computed in partner_contact_birthdate instead of recompute it ? Is it because it's not stored ?

@cvinh @vehi-invitu They failed in some cases like the one below.

**Customer:**

- **Birthdate** = 20-Mar-2007
- **Current Date** = 19-Mar-2025
- In the POS screen, the age displays as **17y**

**Issue:**

If the session is closed on **21-Mar-2025** or later, the age should update to **18y** on **20-Mar-2025.**
However, if the POS screen is not refreshed, it will still display **17y**, even after the birthday has passed.

**Conclusion:**

**res_partner.age** does not correctly handle cases where the session is not closed daily, leading to outdated age calculations on the POS screen

Commit:
[ADD] pos_partner_vat_valid: New module pos_partner_vat_valid

antoniodavid committed:
[IMP] pos_margin: pre-commit execution

antoniodavid committed:
[MIG] pos_margin: Migration to 17.0

Commit:
[UPD] Update pos_partner_vat_valid.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

Commit:
[UPD] Update pos_margin.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: pos-17.0/pos-17.0-pos_partner_vat_valid
Translate-URL: https://translation.odoo-community.org/projects/pos-17-0/pos-17-0-pos_partner_vat_valid/it/

cvinh commented:
@parvezqureshi can you please rebase (as runboat is stuck)

parvezqureshi commented:
> @parvezqureshi can you please rebase (as runboat is stuck)

Rebased

cvinh reviewed (changes requested):
Thanks for the PR.
Here's are steps that do not work as expected (warning is bypassed) :
- the partner does not have a birthdate
- we edit the partner and add the birthdate.
- we save the partner and it's immediatly selected in the pos order without the warning

cvinh commented:
I just realize that this module should be renamed to pos_customer_age_warning ... or pos_customer_birthdate_age_warning...

pos_customer_birthdate_warning is meant to warn if it's the customer's birthday ... 

WDYT ? @legalsylvain

parvezqureshi committed:
[RENAME] Renamed pos_partner_birthdate_warning to  pos_customer_age_warning

parvezqureshi commented:
supersede https://github.com/OCA/pos/pull/1382
